hedera-node/src/main/java/com/hedera/services/fees/charging/NarratedLedgerCharging.java

```diff
@@ -64,6 +64,9 @@ public class NarratedLedgerCharging implements NarratedCharging {
 	private EntityNum nodeId;
 	private EntityNum payerId;
 
+	public static final EntityNum STAKING_REWARD_FUND_ACCOUNT = EntityNum.fromLong(800L);
```

Comment:
We should define these as `AccountID`'s I think, rather than re-creating `AccountID`'s every transaction.

Reply:
Makes sense ! Changed